specs/projectreuniondeployment/projectreuniondeployment.md

```diff
@@ -0,0 +1,138 @@
+required, where to get it, and where to go for instruction.
+
+# Examples
+
+## GetProjectReunionStatus
+
+This is the means by which an app or program can call to verify Project Reunion is present to a
+minimum version. It should be called once after activation before trying to use Project Reunion
+functionality. This method returns a ProjectReunionStatus object that describes the current
+deployment state of Project Reunion.
+
+```C++
+Windows::ApplicationModel::PackageVersion version{1, 0, 0, 0};
+auto status =
+Microsoft::Management::Deployment::ProjectReunionDeployment::GetProjectReunionStatus(version);
+if (!status.IsOK())
+{
+    ...
+}
+```
+
+## GetProjectReunionPackageLink
+
+Provides a direct install link to deploy the latest version of ProjectReunion. If no OS Updates are
+required and the program or app has permissions to deploy packages, this can directly resolve the
+missing dependency without burdening the user.
+
+```C++
+Windows::Foundation::Uri packageLink = Microsoft::Management::Deployment::ProjectReunionDeployment::GetProjectReunionPackageLink();
+```
+
+## GetProjectReunionInstallInformationLink
+
+Provides a user-friendly link to acquire the latest version of Project Reunion and any OS updates
+that may be required. This can be used to guide the user to where they can resolve the missing
+dependency.
+
+```C++
+Windows::Foundation::Uri installInformationLink =
+Microsoft::Management::Deployment::ProjectReunionDeployment::GetProjectReunionInstallInformationLink();
+```
+
+# API Details
+
+```c#
+namespace Microsoft.Management.Deployment
+{
+    /// Represents the current Deployment status of Project Reunion
+    runtimeclass ProjectReunionDeploymentStatus
+    {
+        /// Returns True if Project Reunion is the desired Version and no updates or packages
+        /// are required.
+        Boolean IsOK { get; };
+
+        // Returns True if Windows Updates are required for Project Reunion.
+        Boolean WindowsUpdatesRequired { get; };
+
+        // Returns True if a Package install is required for Project Reunion.
+        Boolean PackageInstallRequired { get; };
+    }
+
+    /// Used to query deployment information for Project Reunion
+    static runtimeclass ProjectReunionDeployment
+    {
+        /// Returns the current deployment status of Project Reunion and whether it meets the 
+        /// minimum package version specified.
+        static ProjectReunionDeploymentStatus GetStatus(Windows.ApplicationModel.PackageVersion packageVersion);
+
+        /// Returns a URI for the direct install link for the latest Project Reunion Package.
+        static Windows.Foundation.Uri GetProjectReunionPackageLink();
+
+        /// Returns a URI for a user-friendly install instruction for Project Reunion and any
+        /// OS updates that may be required.
+        static Windows.Foundation.Uri GetProjectReunionInstallInformationLink();
```

Comment:
This gets 'link for-human-followup vs the other method returns a link for programmatic-followup. Why the dicrepany in names?

```GetProjectReunionPackageLink```
```GetProjectReunionInstallInformationLink```

i.e. Package vs InstallInformation?

CONSIDER: symmetric names e.g. InstallPackage vs InstallInformation. Or Install vs Information? Or Package vs Interactive?

Reply:
Wasn't intending to have a name discrepancy, but changed "Package" to "InstallPackage" and removed the ProjectReunion from the names since that is in the namespace.